events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -70,6 +73,7 @@
         prizeMoney,
         winnerID = null,
         correctColor = null;
+    var formattedAnswer = null;
```

Comment:
Should this be:

        correctColor = null,
        formattedAnswer = null;